### PR TITLE
Zoom to full chromatograms in feature table

### PR DESCRIPTION
**Pull request by robinschmid**
Check FWHM/full width to show either full chromatograms in table or zoomed in features

MZmine 2 was always showing the full range. I like the zoomed in features - but this does not show the whole chromatograms. Especially when there are more peaks like in GC: 

### Left is current state, right is with full chromatograms 
image

### CODE REVIEW

File: src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeChart.java [hunk 1]
```diff
@@ -86,18 +86,21 @@ public FeatureShapeChart(@NotNull ModularFeatureListRow row, AtomicDouble progre
...
+      var fwhm = bestFeature.getFWHM();
+      var fullWidth = RangeUtils.rangeLength(bestFeature.getRawDataPointsRTRange());
+      var dataRTRange = bestFeature.getRawDataFile().getDataRTRange();
+      var rawMinRt = dataRTRange.lowerEndpoint();
+      var rawMaxRt = dataRTRange.upperEndpoint();
+      // FWHM defines most of the feature / chromatogram
+      if (fwhm != null && !Float.isNaN(fwhm) && fwhm > 0f && fwhm / fullWidth > 0.4) {
+        // zoom on feature
+        defaultRange = new org.jfree.data.Range(Math.max(rt - 3 * fwhm, rawMinRt),
+            Math.min(rt + 3 * fwhm, rawMaxRt));
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeChart.java:98]: Why did you change it from 5*fwhm to 3*fwhm?

Comment (robinschmid) [reply src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeChart.java:98]: :D can change it back

Comment (robinschmid) [reply src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeChart.java:98]: I like it more

Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeChart.java:98]: i like 5 more :D
